src/constructors.jl

```diff
@@ -34,6 +34,14 @@ end
 
 @compat (::Type{NullableArray{T}}){T}(dims::Dims) = NullableArray(T, dims)
 @compat (::Type{NullableArray{T}}){T}(dims::Int...) = NullableArray(T, dims)
```

Comment:
It's needed in order to do things like `NullableArray{Int}(3,2)`.

But a call to `NullableArray{T}(dims::Int...)` can always be replace by `NullableArray{T}((dims::Int...))` to call the other constructor with `::Dims` instead.